content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -260,6 +260,38 @@ problems, the kubelet resets the restart backoff timer for that container.
 [Sidecar containers and Pod lifecycle](/docs/concepts/workloads/pods/sidecar-containers/#sidecar-containers-and-pod-lifecycle)
 explains the behaviour of `init containers` when specify `restartpolicy` field on it.
 
+### Configurable container restart delay
+
+{{< feature-state for_k8s_version="v1.32" state="alpha" >}}
+
+With the alpha feature gate `KubeletCrashLoopBackOffMax` enabled, you can
+reconfigure the maximum delay between container restarts from the default of
+300s (5 minutes). This configuration is set per node using kubelet
+configuration. In your [kubelet
+configuration](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/),
```

Comment:
Wrap a line wherever makes sense, but not in the middle of a link.
Use relative path rather than an absolute URL.

```suggestion
300s (5 minutes). This configuration is set per node using kubelet configuration.
In your [kubelet configuration](/docs/tasks/administer-cluster/kubelet-config-file/),
```